pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/onchistory/ESParticipantIdProvider.java

```diff
@@ -42,4 +43,14 @@ public int getParticipantIdForShortId(String shortId) {
             throw new DsmInternalError("Invalid dsm.participant.participantId for shortId " + shortId);
         }
     }
+
+    /**
+     * Given a participant short ID return a participant ID
+     * @throws DsmInternalError for bad ES behavior
+     * @throws DSMBadRequestException when no participant ID is found for short ID
+     */
+    public Optional<ElasticSearchParticipantDto> getParticipantDataForShortId(String shortId) {
```

Comment:
Considered adding method to verifyExitedPtpAndGetId , but since I wanted to collect ALL exited participants shortIds to display back to user, looked like I had to throw exception here, catch and check for that exception in OncHistoryUploadService to move forward. 
To avoid all that, I ended up just getting full ptp data here and doing what I need to do in OncHistoryUploadService. 
I can change if needed.